.circleci/config.yml

```diff
@@ -0,0 +1,26 @@
+version: 2.1
+
+orbs:
+  onegini-build: onegini/onegini-build@2
+
+shared: &shared
+  component_type: library
+  context: dev-context
+  executor:
+    name: onegini-build/maven-builder
+    tag: "11.0"
+
+filter_all_branches_and_tags: &filter_all_branches_and_tags
+  filters:
+    branches:
+      only: /.*/
```

Comment:
Technically this is superfluous, the default is all branches. Only the tags part is needed for releases but even then, it has not real purpose for releasing, so even that could be removed

Reply:
Removed it